sdk/personalizer/Azure.AI.Personalizer/src/Models/RlObjectConverter.cs

```diff
@@ -17,7 +17,7 @@ internal static class RlObjectConverter
         /// </summary>
         public static string ConvertToContextJson(IEnumerable<object> contextFeatures, List<PersonalizerRankableAction> rankableActions)
```

Comment:
What's passing in this `IEnumerable<object>`? That's what has to change. The public API should not expose just `object` anywhere.

Reply:
modified to BinaryData list